squash-the-creeps/rust/src/main_scene.rs

```diff
@@ -0,0 +1,85 @@
+use crate::mob;
+use crate::player;
+use crate::scorelabel;
```

Comment:
Unnecessary import (we import UserInterface via `use crate::scorelabel::UserInterface;` directly)